pkg/hintrunner/zero/zerohint_math_test.go

```diff
@@ -555,5 +555,79 @@ func TestZeroHintMath(t *testing.T) {
 				errCheck: errorTextContains("outside of the range [0, 2**250)"),
 			},
 		},
+
+		"SplitFelt": {
+			{
+				operanders: []*hintOperander{
+					{Name: "MAX_HIGH", Kind: apRelative, Value: &utils.FeltMax128},
+					{Name: "MAX_LOW", Kind: apRelative, Value: &utils.FeltZero},
+					{Name: "low", Kind: uninitialized},
+					{Name: "high", Kind: uninitialized},
```

Comment:
`low` and `high` are supposed to be references to `range_check_ptr` builtin. In the code of the compiler it looks like this:
```
 // Guess the low and high parts of the integer.
let low = [range_check_ptr];
let high = [range_check_ptr + 1];
let range_check_ptr = range_check_ptr + 2;
```

See `Assert250bits` implementation to see how this is handled

Reply:
I have noticed, that in the testcase above I did it correctly, so that must be some copypaste typo. Didn't catch that however.